share/transformers/org_swbiodiversity.py

```diff
@@ -90,10 +93,25 @@ def unwrap_data(self, input_data):
                     contact_dict = {}
                     contact = entry.replace('Contact:', '').strip()
                     contact_email = contact[contact.find("(") + 1:contact.find(")")]
+                    if re.match(r"[^@]+@[^@]+\.[^@]+", contact_email):
+                        if '/' in contact_email:
+                            contact_email = contact_email.split('/')[0]
+                        if ',' in contact_email:
+                            contact_email = contact_email.split(',')[0]
+                    else:
+                        contact_email = None
+
                     contact_name = contact.split('(', 1)[0].strip()
-                    if ', Curator' in contact_name:
-                        contact_name = contact_name.replace(', Curator', '').strip()
-                    if contact and contact_email and re.match(r"[^@]+@[^@]+\.[^@]+", contact_email):
+                    remove_list = ['Curator', 'Science Division Chair', 'Curator', 'Collections Manager',
+                                   'Collections Manager', 'Administrative Director', 'Director', 'Director and Curator',
+                                   'Mycologist and Director', ',']
+                    for item in remove_list:
+                        if item.lower() in contact_name.lower():
+                            contact_name = contact_name.lower().replace(item.lower(), '').strip()
```

Comment:
This means `contact_name` will be lower case if anything is replaced. Could you use a case-insensitive regex instead?